tools/offline_ota_image_builder/README.md

```diff
@@ -0,0 +1,173 @@
+# Offline OTA image builder
+
+`tools.offline_ota_image_builder` is a helper package for building offline OTA image from several input OTA images. Built image can be used as external cache source for otaproxy, or used by full offline OTA helper util to trigger fully offline OTA update.
+
+**NOTE**: full offline OTA helper util is not yet implemented.
+
+It provides the following features:
+
+1. build offline OTA image with several input OTA images,
+2. export built image as tar archive,
+3. prepare and export built image onto specific block device as external cache source dev.
```

Comment:
`external cache source dev` is source device? (what is `dev`?)

Reply:
Ah the word `external cache source` is the terminology from the [OTA cache mechanism design documentation](https://tier4.atlassian.net/l/cp/iSPEDWhT), which means cache source that can be used by otaproxy.
`dev` here means `device`, especially stands for block device. I should not use abbreviation here.

What about the following updated one:
`prepare specific block device, export built image onto it and turns it into an external cache source device(for otaproxy)`.